repobee_feedback/feedback.py

```diff
@@ -111,6 +112,53 @@ def command(self, api: plug.PlatformAPI):
         callback(self.args, api)
 
 
+class GenerateMultiIssueFile(plug.Plugin, plug.cli.Command):
+    __settings__ = plug.cli.command_settings(
+        help="auto generate multi-issue file",
+        description="Will generate a multi-issue file template "
+        "where each pair of student and assignment (args) passed "
+        "will become an issue that starts with the line "
+        "#ISSUE#<STUDENT_REPO_NAME>#<ISSUE_TITLE>, followed by its "
+        "body. Title and body should be filled appropriately later.",
+        category=plug.cli.CoreCommand.issues,
+        action="generate-multi-issue-files",
```

Comment:
As we want to test this plugin, we should replace this with an explicit `Action`. It hasn't historically been used explicitly, so it's a bit hidden.

You can define an action like so:

```python
from repobee_plug.cli.categorization import Action

GENERATE_MULTI_ISSUES_FILE_ACTION = Action(
    name="generate-multi-issues-file",
    category=plug.cli.CoreCommand.issues,
)
```

Then you replace both `category` and `action` liike so:

```suggestion
        action=GENERATE_MULTI_ISSUES_FILE_ACTION,
```

We can use this in testing later.